src/libraries/Microsoft.PowerFx.Core/Public/ErrorKind.cs

```diff
@@ -35,6 +35,8 @@ public enum ErrorKind
         Network = 23,
         Numeric = 24,
         InvalidArgument = 25,
-        Internal = 26
+        Internal = 26,
+        ConnectorWarning = 27,
+        ConnectorError = 28
```

Comment:
Do we need to add new ones - What errors does Power Apps use?

Reply:
PA uses Network
Fixed